### PR TITLE
use complain in setuphost function instead of exit

### PR DESCRIPTION
**Pull request by jdsn**
make a failure more prominent by using the complain function on failure

### CODE REVIEW

**Comment by bmwiedemann** [comment]
:+1:

File: scripts/mkcloud [hunk 1]
```diff
@@ -440,7 +440,7 @@ function setuphost()
         zypper --non-interactive in --no-recommends \
             libvirt kvm lvm2 curl wget bridge-utils \
             dnsmasq netcat-openbsd ebtables libvirt-python
-        [ "$?" == 0 -o "$?" == 4 ] || exit 10
+        [ "$?" == 0 -o "$?" == 4 ] || complain "setuphost failed to install required packages" 10
```
Comment (vuntz) [inline scripts/mkcloud]: Should be `complain 10 "..."`

Comment (jdsn) [inline scripts/mkcloud]: @vuntz Merci

**Comment by vuntz** [comment]
+1

**Pull request merged**